### DISCUSSION TITLE
Hide/Don't Show ENV in Workflow Summary

### DISCUSSION THREAD

**Discussion by woz-94** [community/community#60772] [Actions]
### Select Topic Area

Question

### Body

I have a few secrets which get saved as outputs which then get used as environment variables within steps. After the workflow runs within the summary you can view logs of the run, the shell as well as the environment variables. Because I have a few environment variables which are secrets I was wondering if its possible to disable showing those environment variables. 

![Screenshot 2023-07-13 at 9 40 44 AM](https://github.com/community/community/assets/137090830/73629b9a-97dd-43dd-aa6b-bfcea229648d)

**Comment by jbw9964** [comment]
@woz-94 

Have you solved the problem?
I'm in the same situation.
